src/main/java/com/gd/clinic/service/PrescriptionService.java

```diff
@@ -0,0 +1,59 @@
+package com.gd.clinic.service;
+
+import com.gd.clinic.entity.Prescription;
+import com.gd.clinic.entity.Treatment;
+import com.gd.clinic.repository.PrescriptionRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class PrescriptionService {
+
+    private final PrescriptionRepository prescriptionRepository;
+
+    public Prescription getById(UUID id) throws RuntimeException {
+        Optional<Prescription> fetchedPrescription =  prescriptionRepository.findById(id);
+        if(fetchedPrescription.isPresent()) {
```

Comment:
the same here, please you method of `Optional`

Reply:
Yes, will use `orElseThrow(() ->)` from now on. Corrected it this commit.